vnext/PropertySheets/ReactDirectories.props

```diff
@@ -15,14 +15,17 @@
 
     <ReactNativePackageDir Condition="'$(ReactNativePackageDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'node_modules\react-native\package.json'))\node_modules\react-native\</ReactNativePackageDir>
 
+    <RootDir>$(ReactNativeWindowsDir)</RootDir>
```

Comment:
It is better to have names started with React to avoid any conflict with names that user code or VS code has.

Reply:
I researched the already defined MSVC properties.
These don't collide with those.
It's fair to ask users using this framework to chose their own custom properties.
Moreover, it's highly unlikely that React Native developers would have an issue with MSBuild property names.